.github/workflows/continous-integration-docker.patch.yml

```diff
@@ -93,8 +93,8 @@ jobs:
     steps:
       - run: 'echo "No build required"'
 
-  generate-checkpoints-mainnet:
-    name: Generate checkpoints mainnet / Run generate-checkpoints-mainnet test
+  gen-checkpoints-mainnet:
```

Comment:
Optional: this is 2 characters longer than the next longest ID `sync-past-checkpoint`, so I'd prefer `checkpoints-mainnet`.

(I don't think the `gen-` provides enough important meaning to justify failing on branch names with an extra 2 characters.)

If you decide to make this change, please do it for testnet and the other workflow as well.

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/7643/commits/35bdfd21c1f13ee43b4aaffd8dc9608255d198ae.